drafts/draft-weis-ippm-ioam-eth.xml

```diff
@@ -498,12 +500,16 @@
       endpoints.</t>
     </section>
 
-    <section anchor="IANA" title="IANA Considerations">
+    <section anchor="IANA" title="IEEE Considerations">
```

Comment:
Probably better to keep IANA in the title, e.g. "IANA / IEEE Considerations"

From what I understand, Ethertype is an IANA registry - but managed by the IEEE
Registration Authority  (see e.g. https://www.iana.org/assignments/ieee-802-numbers/ieee-802-numbers.xhtml).
So the section is indeed about an IANA registry.

Reply:
I left it as it is for now. We can clarify this in the later revision.